src/main/java/frc/robot/arm/RealArmIO.java

```diff
@@ -0,0 +1,49 @@
+package frc.robot.arm;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.Encoder;
+import frc.robot.Constants;
+
+public class RealArmIO implements IArmIO {
+
+    private CANSparkMax motorOne;
+    private CANSparkMax motorTwo;
+    private Encoder encoder;
+    private DigitalInput homeSwitch;
+
+    public RealArmIO() {
+        motorOne = new CANSparkMax(Constants.Arm.ARM_MOTOR_ONE, MotorType.kBrushless);
+        motorTwo = new CANSparkMax(Constants.Arm.ARM_MOTOR_TWO, MotorType.kBrushless);
```

Comment:
These should be named `armMotorLeft` and `...Right` with respect to the robot (facing the same direction as the robot, left is on the left). They should amtch the (renamed) constants used.